src/app/lhc-item-gird/lhc-item-gird.component.html

```diff
@@ -0,0 +1,51 @@
+<div *ngIf="item" class="lhc-layout-grid lf-table-item {{lhcDataService.getSiblingStatus(item)}}">
+  <div class="lf-form-grid-table-title" *ngIf="!formLevel">
+    <lhc-item-question-text [item]="item"></lhc-item-question-text>
+  </div>
+
+  <table class="lf-form-grid-table lf-form-table">
+    <colgroup>
+      <col class="lf-question">
+      <col *ngFor="let answer of item.items[0].answers">
```

Comment:
The number of columns should be from item.items, right? The "other-answer" is not needed. Your implementation looks similar to the "matrix" ("table in fhir itemControl). If that's what you need, "matrix" has been implemented in lforms.

There was discussion on chat.fhir.org (https://chat.fhir.org/#narrow/stream/179255-questionnaire/topic/Grid.20Question) that the items under each group are not necessary same. The grid could be sparse.